src/cf.data.pre

```diff
@@ -2364,6 +2364,9 @@ DOC_START
 				Don't request client certificates
 				immediately, but wait until acl processing
 				requires a certificate (not yet implemented).
+                            CONDITIONAL_AUTH
+                                Do not fail auth if client certificate is
```

Comment:
Please see if you can keep the weird formatting consistent by _copying_ the DELAYED_AUTH blob above and preserving copied leading spacing in the new blob. I am sorry we have to do this. Someday, somebody will convert `cf.data.pre` to a more developer-friendly format.